JASP-Desktop/analysis/analysisform.cpp

```diff
@@ -899,6 +902,9 @@ QString AnalysisForm::metaHelpMD() const
 
 QString AnalysisForm::helpMD() const
 {
+	if (!PreferencesModel::prefs()->generateMarkdown())
+		return ""; // It should not come here, but when debugging QML, it crashes here because apparently _analysis is not set yet. Weird...
```

Comment:
Hmm, so it would still crash if the pref isn't set?

And now the property is broken forever whenever you disable the option...
I guess we can leave it for now though.